internal/manager/manager.go

```diff
@@ -53,14 +53,14 @@ var (
 	})
 	scheme               = runtime.NewScheme()
 	setupLog             = ctrl.Log.WithName("setup")
-	enableLeaderElection = managerFlags.Bool("enable-leader-election", false, "Enable leader election for controller manager. "+
+	enableLeaderElection = managerFlags.Bool("leader-elect", false, "Enable leader election for controller manager. "+
```

Comment:
Why change those flags? it brings breaking changes.

Reply:
it's a part of v2-> v3 upgrade

Reply:
Could you mention all the breaking changes in the changelog, and provide update notes?

Reply:
I'm not happy with this change. `enableLeaderElection` is commonly set flag and it ll break a lot of operator deployments. 

 I'd like to postpone those change and introduce it with operator v1 release.

Reply:
@f41gh7 what is a problem with it? people still rely on helm chart, where this flag can be changed without any issues for them. also to support old and new operator versions in helm chart i could add there a condition, which will sets flag names depending on a version

Reply:
My concern is:
- version of chart and operator could managed separately. And you cannot simply rollback chart to previous image version
- it's not reliable to detect operator version by image tag
- not everyone uses helm chart for operator deployment

 Maybe it's fine to mention it at changelog. Until v1 version operator doesn't guarantee backward compatibility.

Reply:
Lets change it, without it migration would be completed.